src/Workspaces/Core/Portable/ProjectTelemetry/ProjectTelemetryInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.ProjectTelemetry
+{
+    internal struct ProjectTelemetryInfo
```

Comment:
`IEquatable<ProjectTelemetryInfo>`?

Reply:
No real point.  This is just a basic serialization type.